### PR TITLE
Added Gittyup.app v1.1.1

### PR DESCRIPTION
**Pull request by HNKNTA**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

**Review by SMillerDev — CHANGES_REQUESTED** [review]
Please add a [`zap` stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap) here. The [CreateZap](https://github.com/nrlquaker/homebrew-createzap) tool can help you with this.

**Comment by HNKNTA** [comment]
> Please add a [`zap` stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap) here. The [CreateZap](https://github.com/nrlquaker/homebrew-createzap) tool can help you with this.

Done, sorry for the inconvenience.

File: Casks/gittyup.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  version "1.1.1"
+  sha256 "d73f876fee2eed4fbbcbc309fa57101332b572f60b648e6c176b04e336f7b03a"
+
+  url "https://github.com/Murmele/Gittyup/releases/download/stable/Gittyup-#{version}.dmg",
+      verified: "github.com/Murmele/Gittyup/"
+  name "gittyup"
+  desc "Graphical Git client"
+  homepage "https://murmele.github.io/Gittyup/"
+
+  livecheck do
```
Comment (SMillerDev) [inline Casks/gittyup.rb:11]: What's wrong with the default livecheck?

Comment (HNKNTA) [reply Casks/gittyup.rb:11]: There are old tags there, the tool is a fork of non-actively developed [Gitahead](https://github.com/gitahead/gitahead).
https://github.com/Murmele/Gittyup/tags

**Review by SMillerDev — APPROVED** [review]
Thanks @HNKNTA  ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Pull request merged**